scripts/cliff.toml

```diff
@@ -56,11 +62,16 @@ commit_parsers = [
     { message = "^refactor", group = "Refactor"},
     { message = "^test", group = "Testing"},
     { message = "^chore", group = "Miscellaneous Tasks"},
+    { message = "^ci", group = "Build System"},
     { message = "^build", group = "Build System"},
-    { message = "^Revert", group = "Reverted Changes"},
+    { message = "^[rR]evert", group = "Reverted Changes"},
+    { message = "^vim-patch", group = "Vim Patches"},
+
+    # All other unclassified commits should be reviewed manually
+    { message = "^.*", group = "(unclassified; TODO: review manually)"},
 ]
-# filter out the commits that are not matched by commit parsers
-filter_commits = true
+# do not filter out the commits that are not matched by commit parsers
+filter_commits = false
```

Comment:
let's move this next to `filter_unconventional`

Reply:
is this change actually needed? wouldn't the `message = "^.*"` commit parser match everything?